### ISSUE TITLE
Incorrect Indention level

### ISSUE THREAD

**Issue by Mottie** [ben-eb/perfectionist#7]
If the css is wrapped in an `@` rule, the resulting indention & breaks between rules are incorrect:

Starting with this css:

```css
@-moz-document domain('test.com') {
  .selector 1, .selector 2, .selector 3, .selector 4, .selector 5, .selector 6, .selector 7, .selector 8, .selector 9, .selector 10, .selector 11, .selector 12, .selector 13, .selector 14, .selector 15, .selector 16, .selector 17, .selector 18, .selector 19, .selector 20 {
    background: #000;
  }
}

@media all and (min-width: 480px) and (max-width: 767px) {
  .selector 1, .selector 2, .selector 3, .selector 4, .selector 5, .selector 6, .selector 7, .selector 8, .selector 9, .selector 10, .selector 11, .selector 12, .selector 13, .selector 14, .selector 15, .selector 16, .selector 17, .selector 18, .selector 19, .selector 20 {
    background: #000;
  }
}
```

with default settings, the result becomes:

```css
@-moz-document domain('test.com') {
    .selector 1, .selector 2, .selector 3, .selector 4, .selector 5, .selector 6,
.selector 7, .selector 8, .selector 9, .selector 10, .selector 11, .selector 12,
.selector 13, .selector 14, .selector 15, .selector 16, .selector 17,
.selector 18, .selector 19, .selector 20 {
        background: #000;
    }
}@media all and (min-width: 480px) and (max-width: 767px) {
    .selector 1, .selector 2, .selector 3, .selector 4, .selector 5, .selector 6,
.selector 7, .selector 8, .selector 9, .selector 10, .selector 11, .selector 12,
.selector 13, .selector 14, .selector 15, .selector 16, .selector 17,
.selector 18, .selector 19, .selector 20 {
        background: #000;
    }
}
```

Also, is there a way to set the indention to use 2 spaces instead of 4?

**Comment by ben-eb** [comment]
Yep, can probably add some extra space there too. :+1:

**Comment by ben-eb** [comment]
Released as `1.3.0`.

**Comment by Mottie** [comment]
Excellent work! :cookie:
